### DISCUSSION TITLE
Simplify committee details

### DISCUSSION THREAD

**Discussion by hturner** [WarwickRUG/WarwickRUG.github.io#18] [General]
The new About page us page has a separate page for each committee member. I'm not sure that people will want to spend that much time clicking through to each page. Maybe we can modify the photo gallery to give a little more info about each person up front? E.g. Position, Affiliation, links - something like https://contributor.r-project.org/r-project-sprint-2023/participants/. This would be easier to maintain and if we put some sort of profile link (e.g. personal webpage/GitHub profile/LinkedIn profile), then people can get more information if interested.

**Comment by ccamara** [comment]
I honestly believe that the current implementation is quite good and not complex (although it could be improved): we just need to add a `qmd` file in the `people/` folder, and the list will be populated from the file's title (person's name) and image. If no image is provided, a default one will be used. This way, there's only a single source of truth (the person's file).

I agree that some people may not want to update that profile, though. In that case, we could tweak the list from grid to fields and display only certain fields, such as title, image and links items as described here https://quarto.org/docs/websites/website-listings.html#listing-fields (that said, I do not know how to show nested items such as those in links). Another option to consider could be creating a custom listing (https://quarto.org/docs/websites/website-listings-custom.html) and combine that with CSS rules to make it look nice. I haven't tried that either, but seems doable.

That said, a static page could also work, provided how little this is going to change over time.

**Reply by hturner** [reply]
Ah yes, the implementation is neat, I hadn't looked into the detail to see that you were getting the biographic information from the YAML header (e.g. of https://github.com/WarwickRUG/WarwickRUG.github.io/edit/main/people/heather-turner.qmd). I think a custom listing is a good idea, so that the key info is on the About page and a link either goes to the full bio (which may in fact not show any further info) or perhaps a custom field (say `profile-link`). It seems from the [gallery](https://quarto.org/docs/gallery/) (which is their example of a custom listing) that the main link can be something external, rather than the source page.

**Reply by ccamara** [reply]
Oh, that (having a link to either an external page or to the bio) would be even better! I've created this issue #24 (it's just a placeholder for now).
